packages/core/components/EditMetadata/ExistingAnnotationPathway.tsx

```diff
@@ -65,16 +81,58 @@ export default function ExistingAnnotationPathway(props: ExistingAnnotationProps
                 ...valueMap,
             ];
         }
-        setSelectedAnnotation(selectedFieldName);
-        setAnnotationType(option?.data);
-        setValueCount(valueMap);
+
+        // String type annotations might be dropdown or lookup
+        // If it's a dropdown, we need to fetch the dropdown options
+        if (option?.data === AnnotationType.STRING) {
+            annotationService
+                .fetchAnnotationDetails(selectedFieldName)
```

Comment:
Is this likely to ever take long (i.e., any chance we'd need a loading indicator on the UI side?)

Reply:
I didn't see any performance slowness in my testing so I'm going to roll with this for now and if we ever notice it start to slow we can add some loading stuff